### PR TITLE
Kouros new changes

### PR DESCRIPTION
**Pull request by aswinssir**
Kouros changes according to new Architecture
D-Agent changes related to Kouros

### CODE REVIEW

File: src/dagent/src/routers/m9k/route.go [hunk 1]
```diff
@@ -36,16 +36,16 @@ import (
 	"dagent/src/routers/m9k/api/caller"
 	"dagent/src/routers/m9k/api/dagent"
 	"dagent/src/routers/m9k/api/ibofos"
-	"dagent/src/routers/m9k/api/ibofos_"
```
Comment (vishal-shakya-ssir) [inline src/dagent/src/routers/m9k/route.go:39]: since we are not using socket connection so we can delete ibofos_ and api_ package and files

**Review by vishal-shakya-ssir — CHANGES_REQUESTED** [review]

File: src/dagent/src/routers/m9k/route.go (hunk 1 above)
Comment (aswinssir) [reply src/dagent/src/routers/m9k/route.go:39]: Thanks for the comment. Removed those files. Please check

**Review by vishal-shakya-ssir — APPROVED** [review]
LGTM

**Review by manoj22-k — APPROVED** [review]
LGTM

**Pull request merged**